bin/install-dependencies.sh

```diff
@@ -1,7 +1,7 @@
 #! /bin/bash
 
 venv/bin/pip install -U $(curl -s https://raw.githubusercontent.com/ckan/ckan/2.7/requirement-setuptools.txt)
-venv/bin/pip install -U $(curl -s https://raw.githubusercontent.com/ckan/ckanext-harvest/v1.1.0/pip-requirements.txt)
+venv/bin/pip install -U $(curl -s https://raw.githubusercontent.com/ckan/ckanext-harvest/c21c1627992cbcf0043b7b6873ff899f6f66767f/pip-requirements.txt)
```

Comment:
Hopefully my version won't break anything (I've tested it locally)

Reply:
Your version of what sorry?

Reply:
@elliotcm this https://github.com/alphagov/ckanext-harvest it's the upstream one (forked yesterday) with the SSL patches the team applied in the last weeks/months. I will try to have them accepted upstream too, but in the mean time, if you want the SSL problem fixed, you can use our fork.

Reply:
@andreagrandi Ah gotcha.  Should be fine to rebase that branch off the upstream commit mentioned here (or cherry-pick @brucebolt's commit), it's only a very tiny change.